pkg/cmd/env/list.go

```diff
@@ -0,0 +1,37 @@
+package env
+
+import (
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	corectlenv "github.com/coreeng/corectl/pkg/env"
+	"github.com/coreeng/developer-platform/pkg/environment"
+	"github.com/spf13/cobra"
+)
+
+func listCmd(cfg *config.Config) *cobra.Command {
+	listCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List all environments",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return list(cfg)
+		},
+	}
+	return listCmd
+}
+
+func list(cfg *config.Config) error {
+	if err := config.NotExist(); err != nil {
```

Comment:
Other commands allow to have no config, but provide all the necessary values from the flags.
Here is an example: https://github.com/coreeng/corectl/blob/980ff32f335e82aa369acbb357c1772a0ecd9daa/pkg/cmd/application/create/app_create.go#L74

If we follow this pattern, we can drop `config.NotExist()` function

Also, let's check for the local repository to be present and be in the correct state is it's done here: https://github.com/coreeng/corectl/blob/980ff32f335e82aa369acbb357c1772a0ecd9daa/pkg/cmd/application/create/app_create.go#L95

Reply:
You can have a flag to specify the location of the repository
I should've mentioned it, sorry